### PR TITLE
[Deploy] Avoid infinite loop while waiting for image streams

### PR DESCRIPTION
**Pull request by avano**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java [hunk 1]
```diff
@@ -237,9 +237,15 @@ private static void deploySyndesisViaOperator() {
 
     private static void patchImageStreams() {
         ImageStreamList isl = OpenShiftUtils.client().imageStreams().inNamespace(TestConfiguration.openShiftNamespace()).withLabel("syndesis.io/component").list();
+        final int maxRetries = 120;
```
Comment (tplevko) [inline utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: @avano this is probably something I'll need more info about.

File: utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java [hunk 2]
```diff
@@ -237,9 +237,15 @@ private static void deploySyndesisViaOperator() {
...
+        int retries = 0;
         while (isl.getItems().size() < 6) {
```
Comment (tplevko) [inline utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: didn't the default image stream count increased to 7?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**